internal/controller/policyexceptiondraft_controller.go

```diff
@@ -18,13 +18,13 @@ package controller
 
 import (
 	"context"
-
 	"fmt"
 
-	giantswarmExceptions "github.com/giantswarm/exception-recommender/api/v1alpha1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	giantswarmPolicy "github.com/giantswarm/exception-recommender/api/v1alpha1"
```

Comment:
Consider: this introduces a direct dependency, so you'll need to couple the releases in the future

Reply:
Actually, since we won't use the Drafts, will host `PolicyException` directly in this repo and this shouldn't be needed anymore.

Reply:
Fine for now, PolicyExceptions will eventually need a different home

Reply:
I would like to have them all in a same place to be honest. PolicyExceptions, Drafts, and so on

Reply:
Yes for sure